server/src/main/java/tabletop/repositories/TournamentFinalResultRepository.java

```diff
@@ -0,0 +1,15 @@
+package tabletop.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import tabletop.domain.match.tournament.Tournament;
+import tabletop.domain.match.tournament.TournamentFinalResult;
+import tabletop.domain.user.User;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface TournamentFinalResultRepository extends JpaRepository<TournamentFinalResult, Long> {
+    List<TournamentFinalResult> findByTournamentOrderByPlace(Tournament tournament);
+    Optional<TournamentFinalResult> findOneByUserAndTournament(User user, Tournament tournament);
```

Comment:
at first I also used optionals in repositories, then something made me think it only works with nulls so I changed, but I see that I was wrong :P so I will also change these in event and location repository